mavis.wdl

```diff
@@ -1,264 +1,228 @@
+    String? mavisScheduler = "SGE"
+    String? mavisDrawFusionOnly = "False"
+    Int? mavisAnnotationMemory = 32000
+    Int? mavisValidationMemory = 32000
+    Int? mavisTransValidationMemory = 32000
+    Int? mavisMemoryLimit = 32000
+    Int? minClusterPerFile = 5
+    String? drawNonSynonymousCdnaOnly = "False"
+    String? mavisUninformativeFilter = "True"
+    String? modules = "mavis/2.2.6 hg19-mavis/2.2.6 hg19/p13"
+    Int?   jobMemory = 12
+    Int?   sleepInterval = 20
+  }
+
+  parameter_meta {
+    inputBAMs: "array of input bam files"
+    inputBAMidx: "array of input .bai files"
+    svData: "array of SV calls"
+    libTypes: "List of library types, metadata for inputBAMs"
+    svWorkflows: "List of SV callers, metadata for svData"
+    svLibDesigns: "List of library designs to accompany the list of SV calls"
+    outputCONFIG: "name of config file for MAVIS"
+    scriptName: "name for bash script to run mavis configuration, default mavis_config.sh"
+    donor: "donor id, i.e. PCSI_0001 Identifies a patient, cell culture grown at certain condition etc."
+    referenceGenome: "path to fasta file with genomic assembly"
+    annotations: ".json file with annotations for MAVIS"
+    masking: "masking data in .tab format"
+    dvgAnnotations: "The DGV annotations help to deal with variants found in normal tissue"
+    alignerReference: "References in 2bit (compressed) format, used by MAVIS aligner"
+    templateMetadata: "Chromosome Band Information, used for visualization"
+    mavisAligner: "blat by default, may be customized"
+    mavisScheduler: "Our cluster environment, sge, SLURM etc. "
+    mavisDrawFusionOnly: "flag for MAVIS visualization control"
+    mavisAnnotationMemory: "Memory allocated for annotation step"
+    mavisValidationMemory: "Memory allocated for validation step"
+    mavisTransValidationMemory: "Memory allocated for transvalidation step"
+    mavisMemoryLimit: "Max Memory allocated for MAVIS"
+    minClusterPerFile: "Determines the way parallel calculations are organized "
+    drawNonSynonymousCdnaOnly: "flag for MAVIS visualization control"
+    mavisUninformativeFilter: "Should be enabled if used is only interested in events inside genes, speeds up calculations"
+    modules: "modules needed to run MAVIS"
+    jobMemory: "Memory allocated for this job"
+    sleepInterval: "A pause after scheduling step, in seconds"
+  }
+
+  command <<<
+    unset LD_LIBRARY_PATH
+    unset LD_LIBRARY_PATH_modshare
+    export MAVIS_REFERENCE_GENOME=~{referenceGenome}
+    export MAVIS_ANNOTATIONS=~{annotations}
+    export MAVIS_MASKING=~{masking}
+    export MAVIS_DGV_ANNOTATION=~{dvgAnnotations}
+    export MAVIS_ALIGNER_REFERENCE=~{alignerReference}
+    export MAVIS_TEMPLATE_METADATA=~{templateMetadata}
+    python <<CODE
+
+    libtypes = {'WT': "transcriptome", 'MR': "transcriptome", 'WG': "genome"}
+    wfMappings = {'StructuralVariation': 'delly', 'delly': 'delly', 'StarFusion': 'starfusion', 'manta': 'manta'}
```

Comment:
StarFusion -> starFusion (to match the artifact id of the released starFusion workflow)